tests/sanity.py

```diff
@@ -0,0 +1,194 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import json
+import logging
+import os
+
+import boto.ec2
+from retrying import retry
+from fabric.api import settings as fabric_settings
+from fabric.api import run as fabric_run
+
+
+REGION = 'us-east-1'
+CENTOS_7_AMI_ID = 'ami-96a818fe'
```

Comment:
we are logging this ami public ip, are we certain it does not have password ssh enabled?